docs/nv2/registry/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM notaryv2/registry:nv2-prototype-2
+
+RUN apk add openssl
+
+COPY entrypoint.sh /
```

Comment:
seems like we are missing a chmod +x here...
without it , the image wouldn't start correctly @sudo-bmitch

Reply:
The +x is already on the file in the source. I don't think it's needed. Are you seeing errors?

Reply:
This may impact windows desktop users not in WSL2, so I added it.